### PR TITLE
add integration test for the install script

### PR DESCRIPTION
**Pull request by davidgamero**
# Description
The current script does not install draft and returns a 404
This PR corrects the path to the raw script from this repo
Also added an integration test to validate the functionality of the install script added in #147

Fixes # (issue)
Feature # (details)

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration. Is it a  breaking change which will impact consuming tool(s).

- [x] Existing tests pass
- [x] Added a new integration test that validates the install completes

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: .github/workflows/integration-install.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+jobs:
+  install:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Run Install Script 
+        run: ./scripts/install.sh
+      - name: Validate Command is Installed
+        run: |
+          if ! command -v draft &> /dev/null
```
Comment (Tatsinnit) [inline .github/workflows/integration-install.yml:14]: 💡 Looks good and great to see check working : https://github.com/Azure/draft/actions/runs/3231609092/jobs/5291313431 

just minor fyi - I wonder how `draft` get set into `PATH` but all good as this clearly works.

**Review by Tatsinnit — APPROVED** [review]
☕️🎉🤟 looks good. thanks.

File: .github/workflows/integration-install.yml (hunk 1 above)
Comment (davidgamero) [reply .github/workflows/integration-install.yml:14]: the github action runner PATH automatically includes the .local/bin directory 😄

Comment (davidgamero) [reply .github/workflows/integration-install.yml:14]: i can't find a doc for this, but it does work!

**Review by Tatsinnit — APPROVED** [review]

**Review by OliverMKing — APPROVED** [review]

**Review by OliverMKing — CHANGES_REQUESTED** [review]

**Review by OliverMKing — APPROVED** [review]

**Review by tbarnes94 — APPROVED** [review]

**Pull request merged**